lib/oli_web/components/delivery/instructor_dashboard.ex

```diff
@@ -53,14 +58,17 @@ defmodule OliWeb.Components.Delivery.InstructorDashboard do
 
   def tabs(assigns) do
     ~H"""
-      <div class="container mx-auto my-4">
-        <ul class="nav nav-tabs flex flex-col md:flex-row flex-wrap list-none border-b-0 pl-0 mb-4" id="tabs-tab"
-          role="tablist">
-
-          <%= for %TabLink{label: label, path: path, badge: badge, active: active} <- @tabs do %>
-            <li class="nav-item" role="presentation">
-              <.link patch={path}
-                class={"
+    <div class="container mx-auto my-4">
+      <ul
+        class="nav nav-tabs flex flex-col md:flex-row flex-wrap list-none border-b-0 pl-0 mb-4"
+        id="tabs-tab"
+        role="tablist"
+      >
+        <%= for %TabLink{label: label, path: path, badge: badge, active: active} <- @tabs do %>
```

Comment:
NIT: This looks like a good opportunity to replace that `<%= for... do %>` for LiveView's new `:for` decorator 😁